### DISCUSSION TITLE
How do I match text between <> brackets using regexes?

### DISCUSSION THREAD

**Discussion by k-soomro** [typst/typst#4652] [General]
I am using the following regex:

    #show regex("<.+>"): highlight

But it doesn't work. A single `<` correctly matches angle brackets but as soon as I type any other character it doesn't match anything. For example, I have the following text:

    \<20...

But the regex `<20` doesn't match this either.

Escaping the brackets doesn't work either.

## MWE

```typst
#show regex("<.*>"): highlight

\<#lorem(10)\>
```

**Comment by Mc-Zen** [comment]
Hi, unfortunately this is not possible at the moment. The underlying reason is that regexes don't match across text nodes and in your case, 
`\<20` becomes two text nodes `[<]` and `[20]`. The relevant issue here on GitHub is #86 . As far as I know there are plans to make this possible, however, this might still take some time.

**Reply by k-soomro** [reply]
Thanks. Might I suggest that this should be documented in the show rule entry in the docs until it's fixed? I spent quite a bit of time trying to figure out why this wasn't working.
